misc/python/materialize/feature_benchmark/benchmark.py

```diff
@@ -72,7 +72,7 @@ def run(self) -> list[Aggregation]:
 
         # Run the shared() section once for both Mzs under measurement
         shared = scenario.shared()
-        if self._mz_id == 0 and shared is not None:
+        if shared is not None:
```

Comment:
Can you explain this change?

Reply:
Previously we ran the shared section once, running something like the Kafka commands, then run the benchmark using this Kafka data against old and new versions.

Now the shared section is always run, and we reset the Kafka state. But actually let me try without that and keeping the shared state.